containers/server-image/Dockerfile

```
@@ -4,10 +4,12 @@
 ARG INIT_IMAGE=uyuni/init:latest
 FROM $INIT_IMAGE
 
-ARG PRODUCT_PATTERN_PREFIX="patterns-uyuni"
+# Value could also be susemanager
+ARG PRODUCT="uyuni"
 
 # Extra packages can be added via project configuration
-ARG EXTRAPACKAGES
+# syslinux should be syslinux-x86_64 on non-x86_64 architectures
+ARG EXTRAPACKAGES=syslinux
```

Considering that we already pass some  EXTRAPACKAGES  via prjconf I would  continue  to specify `syslinux`  there given  that the  package  name  is  not called the same way in all  the architectures. 
```suggestion
ARG EXTRAPACKAGES
```

---

currently running  some tests

---

This is just a default value. I can remove it for sure